Gemfile

```diff
@@ -64,7 +64,6 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of chromedriver to run system tests with Chrome
   gem 'chromedriver-helper'
-  gem 'faker'
```

Comment:
Remove duplicate included gem- it's added again a little farther down in the file and it creates a bundler warning